Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 60
+        "Patch": 61
```

Comment:
Before check-in please make sure the Patch version is appropriate. Do a gulp test once. Looks fine otherwise.